packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FluentIconsProps } from '@fluentui/react-icons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as ReactIcons from '@fluentui/react-icons';
+import { Input } from '@fluentui/react-input';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    width: '100%',
+    marginLeft: 'auto',
+    boxSizing: 'border-box',
+    marginRight: 'auto',
+    display: 'block',
+    paddingLeft: '16px',
+    paddingRight: '16px',
+    paddingTop: '10px',
+    fontFamily: 'Arial',
+    maxWidth: '105ch',
+    position: 'relative',
+
+    '& span': {
+      width: '50%',
+      display: 'inline-block',
+      flexDirection: 'column',
+      margin: '0px 4px',
+
+      '& svg': {
+        display: 'inline-block',
+      },
+
+      // '& div': {
+      //   color: theme.colorNeutralForeground1,
+      //   fontSize: '16px',
+      //   // opacity: '0',
+      // },
+
+      // '&:hover': {
+      //   overflow: 'visible',
+
+      //   '& span': {
+      //     opacity: '1',
+      //   },
+      // },
+    },
+  }),
+
+  searchBox: {
+    maxWidth: '320px',
+  },
+
+  radio: {
+    fontSize: 'small',
```

Comment:
This should be set to a specific size--I'm not sure what `small` even means in this context